src/jit/gentree.cpp

```diff
@@ -14397,7 +14397,7 @@ bool Compiler::gtTreeHasSideEffects(GenTreePtr tree, unsigned flags /* = GTF_SID
     return true;
 }
 
-GenTreePtr Compiler::gtBuildCommaList(GenTreePtr list, GenTreePtr expr)
+GenTreePtr Compiler::gtBuildCommaList(GenTreePtr list, GenTreePtr expr DEBUGARG(bool remorphing))
```

Comment:
Using an argument preserves the original assertion in cases that haven't been explicitly checked. I'm open to changing this to use `fgGlobalMorph` if that precision seems unnecessary,